components/BottomNavigation/src/MDCBottomNavigationBar.m

```diff
@@ -653,4 +651,11 @@ + (NSString *)bundlePathWithName:(NSString *)bundleName {
   return [resourcePath stringByAppendingPathComponent:bundleName];
 }
 
+#pragma mark - MDCInkTouchControllerDelegate methods
+
+- (MDCInkView *)inkTouchController:(MDCInkTouchController *)inkTouchController
+            inkViewAtTouchLocation:(CGPoint)location {
+  return ((MDCBottomNavigationItemView *)inkTouchController.view).inkView;
```

Comment:
Can/should we verify that inkTouchController.view == MDCBottomNavigationItemView?

Reply:
Done